packages/yourloops/pages/hcp/team-card.tsx

```diff
@@ -0,0 +1,182 @@
+/**
+ * Copyright (c) 2021, Diabeloop
+ * Team card for HCPs
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import * as React from "react";
+
+import { makeStyles, Theme } from "@material-ui/core/styles";
+import Avatar from "@material-ui/core/Avatar";
+import Button from "@material-ui/core/Button";
+import Paper from "@material-ui/core/Paper";
+
+import EditIcon from '@material-ui/icons/Edit';
+import EmailIcon from '@material-ui/icons/Email';
+import FingerprintIcon from '@material-ui/icons/Fingerprint';
+import LocationOnIcon from '@material-ui/icons/LocationOn';
+import PersonAddIcon from '@material-ui/icons/PersonAdd';
+import PhoneIcon from '@material-ui/icons/Phone';
+
+import { Team } from "../../models/team";
+import { t } from "../../lib/language";
+
+import TeamEditModal from "./team-edit-modal";
+
+interface TeamCardProps {
+  team: Team;
+  onEditTeam: (team: Team) => Promise<void>;
+}
+
+interface TeamInfoProps {
+  label: string;
+  value?: string;
+  icon: JSX.Element;
+}
+
+const teamCardStyles = makeStyles((theme: Theme) => {
+  return {
+    paper: {
+      display: "flex",
+      flexDirection: "column",
+      backgroundColor: theme.palette.primary.light,
+    },
+    paperRoot: {
+      padding: "1em",
+    },
+    firstRow: {
+      display: "flex",
+      flexDirection: "row",
+      marginBottom: theme.spacing(2), // eslint-disable-line no-magic-numbers
+    },
+    secondRow: {
+      display: "flex",
+      flexDirection: "row",
+      justifyContent: "space-between",
```

Comment:
```suggestion
      justifyContent: "left",
      columnGap: "10px",
```
I know I'm not supposed to play with css ;-(

Reply:
Yes, I know this case.
But It need a little bit more tunning. I was planning fixing it in the last PR of this screen.